service/__init__.py

```diff
@@ -38,7 +38,14 @@ def before_first_request():
     Set price only after all services are running - digital-register-api in particular.
     """
 
-    price = "{0:.2f}".format(api_client.get_pound_price())
+    price = api_client.get_pound_price()
+
+    # If the price is a round number of pounds we don't want to display any decimal places
+    if price.is_integer():
```

Comment:
@russwillis digital-register-frontend repeatedly deviated from the prototype in this respect, so I fixed it... So, I would say this is UR driven :-)